avromatic.gemspec

```diff
@@ -31,8 +31,8 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = '>= 2.7'
 
-  spec.add_runtime_dependency 'activemodel', '>= 5.2', '< 7.2'
-  spec.add_runtime_dependency 'activesupport', '>= 5.2', '< 7.2'
+  spec.add_runtime_dependency 'activemodel', '>= 5.2'
```

Comment:
Let's go with a version constraint of `< 8` so we can ensure compatibility with major rails releases.

Reply:
No worries 👍